test/gitea_custom_console_dashboard_test.go

```diff
@@ -61,21 +44,13 @@ func TestGiteaCustomConsoleDashboard(t *testing.T) {
 	// asserting with non empty because we are not sure the URL for K8S and Openshift
 	assert.Assert(t, prs.Items[0].Annotations[keys.LogURL] != "")
 
-	// update config map with console dashboard information
-	cmList, er := topts.ParamsRun.Clients.Kube.CoreV1().ConfigMaps("").List(ctx, metav1.ListOptions{})
-	assert.NilError(t, er)
-	for _, v := range cmList.Items {
-		if v.Name == "pipelines-as-code" {
-			cmData, err := topts.ParamsRun.Clients.Kube.CoreV1().ConfigMaps(v.Namespace).Get(ctx, v.Name, metav1.GetOptions{})
-			assert.NilError(t, err)
-			cmData.Data[customConsoleDashboardName] = "My custom console"
-			cmData.Data[customConsoleDashboardURL] = "https://custom-console.ospqa.com"
-			cmData.Data[customConsoleDashboardPRDetailURL] = "https://custom-console.ospqa.com/foo-pr"
-			cmData.Data[customConsoleDashboardPRTaskLog] = "https://custom-console.ospqa.com/foo-pr/logs/task"
-			_, err = topts.ParamsRun.Clients.Kube.CoreV1().ConfigMaps(v.Namespace).Update(ctx, cmData, metav1.UpdateOptions{})
-			assert.NilError(t, err)
-		}
+	data := map[string]string{
+		customConsoleDashboardName:        "My custom console",
+		customConsoleDashboardURL:         "https://custom-console.ospqa.com",
```

Comment:
it's bad practice to  output infra specific URL in unittest

Reply:
Updated to "https://custom-console.test.com",